controllers/cluster_controller.go

```diff
@@ -384,21 +416,26 @@ func (r *ClusterReconciler) reconcileDelete(ctx context.Context, log logr.Logger
 	return ctrl.Result{}, nil
 }
 
-func (r *ClusterReconciler) ensureClusterOwnerReferences(ctx context.Context, clus *anywherev1.Cluster) error {
+func (r *ClusterReconciler) getChildObjectsForCluster(ctx context.Context, clus *anywherev1.Cluster) ([]kubernetes.Object, error) {
```

Comment:
this method is a bit terse
I would lean towards returning the cluster.Config like `buildClusterConfig`
And then let the callers use whatever they need from it

Reply:
Done!